sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/CosmosRowConverter.scala

```diff
@@ -223,6 +226,15 @@ private object CosmosRowConverter
       }
     }
 
+  private def parseLsn(objectNode: ObjectNode): Long = {
+    objectNode.get(LsnAttributeName)
+    match {
+      case lsnNode: JsonNode =>
+        Option(lsnNode).fold(-1L)(v => v.asLong(-1))
```

Comment:
Could this Option condition be false if the `lsnNode` is provided?

Reply:
ObjectNode.get(PorpertyName) returns null (JsonNode) whenever a property doesn't exist on the ObjectNode - so this covers the case where the "_lsn" property would be missing.

Scenario where this could happen: custom schema contains "_lsn" of LongType - but DF is actually a query not changefeed